frontend/packages/ceph-storage-plugin/src/components/ocs-install/internal-mode/install-wizard.tsx

```diff
@@ -92,7 +92,7 @@ export const CreateInternalCluster: React.FC<CreateInternalClusterProps> = ({ ma
       component: (
         <ReviewAndCreate state={state} errorMessage={errorMessage} inProgress={inProgress} />
       ),
-      enableNext: hasEnabledCreateStep,
+      enableNext: hasEnabledCreateStep && !state.isArbiterValid,
```

Comment:
```suggestion
      enableNext: hasEnabledCreateStep
```
Why not keep it as a part of `hasEnabledCreateStep`?